File: android/mobilepasssdk/src/main/java/com/armongate/mobilepasssdk/model/DeviceConnectionInfo.java
package com.armongate.mobilepasssdk.model;

public class DeviceConnectionInfo {

    public String deviceId;
    public String publicKey;

    public DeviceConnectionInfo(String id, String publicKey) {
        this.deviceId   = id;
        this.publicKey  = publicKey;
    }

}

File: ios/MobilePassSDK/Model/DeviceConnectionInfo.swift
//
//  DeviceConnectionInfo.swift
//  MobilePassSDK
//
//  Created by Erinc Cakir on 22.04.2021.
//

import Foundation

struct DeviceConnectionInfo {
    var deviceId:   String;
    var publicKey:  String;
    
    init(deviceId: String, publicKey: String) {
        self.deviceId   = deviceId
        self.publicKey  = publicKey
    }
}
